scripts/setup_devmode.sh

```diff
@@ -44,9 +44,10 @@ CFG_TMPDIR = u'/tmp'
 COLLECT_STORAGE = u'flask_collect.storage.link'
 EOF
 
+echo "Building Assets, bower install"
 # fetch and build assets
 inveniomanage bower -i bower-base.json > bower.json
-bower install --silent
+bower install --quiet
```

Comment:
There IS an issue. Right now, you have to download the log anyway to see the full test results.